attributes/default.rb

```diff
@@ -95,6 +95,7 @@
       '/media/downloadable' => 'restricted',
       '/pkginfo' => 'restricted',
       '/report/config.xml' => 'restricted',
+      '^/rss/(catalog/(notifystock|review)|order/new)(/|$)' => 'restricted',
```

Comment:
I think the starting "/" should be optional, as its also accessible without it.
Ex. "index.phprss/catalog/notifystock/"
Ref: https://support.hypernode.com/knowledgebase/how-to-protect-your-magento-store-against-brute-force/